.env.example

```
@@ -29,6 +29,17 @@ RANKING_SCHEMA=
 AI_SCHEMA=
 NOTIFICATION_SCHEMA=
 
+# db schema
+USER_SCHEMA=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The USER_SCHEMA key is duplicated

(DuplicatedKey)